File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/AudioConfig.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class AudioConfig{

	@SerializedName("perceptualLoudnessDb")
	private double perceptualLoudnessDb;

	@SerializedName("loudnessDb")
	private double loudnessDb;

	@SerializedName("enablePerFormatLoudness")
	private boolean enablePerFormatLoudness;

	public void setPerceptualLoudnessDb(double perceptualLoudnessDb){
		this.perceptualLoudnessDb = perceptualLoudnessDb;
	}

	public double getPerceptualLoudnessDb(){
		return perceptualLoudnessDb;
	}

	public void setLoudnessDb(double loudnessDb){
		this.loudnessDb = loudnessDb;
	}

	public double getLoudnessDb(){
		return loudnessDb;
	}

	public void setEnablePerFormatLoudness(boolean enablePerFormatLoudness){
		this.enablePerFormatLoudness = enablePerFormatLoudness;
	}

	public boolean isEnablePerFormatLoudness(){
		return enablePerFormatLoudness;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/AudioConfig.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class AudioConfig : Parcelable, Serializable {
    @SerializedName("perceptualLoudnessDb")
    var perceptualLoudnessDb = 0.0

    @SerializedName("loudnessDb")
    var loudnessDb = 0.0

    @SerializedName("enablePerFormatLoudness")
    var isEnablePerFormatLoudness = false
    override fun toString(): String {
        return "AudioConfig{" +
                "perceptualLoudnessDb = '" + perceptualLoudnessDb + '\'' +
                ",loudnessDb = '" + loudnessDb + '\'' +
                ",enablePerFormatLoudness = '" + isEnablePerFormatLoudness + '\'' +
                "}"
    }
}